contracts/src/challengeV2/EdgeChallengeManager.sol

```diff
@@ -308,9 +338,10 @@ contract EdgeChallengeManager is IEdgeChallengeManager, Initializable {
         uint256 layerZeroBigStepEdgeHeight,
         uint256 layerZeroSmallStepEdgeHeight,
         IERC20 _stakeToken,
-        uint256 _stakeAmount,
         address _excessStakeReceiver,
-        uint8 _numBigStepLevel
+        uint8 _numBigStepLevel,
+        uint256 _initialStakeAmount,
+        uint256 _stakeAmountSlope
     ) public initializer {
```

Comment:
not directly related to this pr but since the challenge manager should be immutable we can remove the initializer and use a constructor so that a brunch of these config can be made immutable to save gas